sycl/include/sycl/ext/oneapi/matrix/matrix-jit.hpp

```diff
@@ -451,6 +451,284 @@ class wi_element {
   }
 };
 
+// uint16_t here represents bf16 type as we have been doing for the
```

Comment:
Some rewording:
// Note that similarly to the other matrix functions, uint16_t is used here to represent bf16 type. Since the AMX and DPAS implementations don't support uint16_t, this interpretation is possible. This design choice was made before the introduction of SYCL experimental bfloat16 type. Our plan is to move towards using the SYCL bfloat16. But since it is still experimental, we will probably keep both uint16 interpretation and SYCL bfloat16.